src/reanimated2/animation/util.ts

```diff
@@ -73,6 +75,15 @@ function recognizePrefixSuffix(value: string | number): RecognizedPrefixSuffix {
   }
 }
 
+export function shouldReduceMotion(config?: ReducedMotionConfig) {
+  'worklet';
+  if (config === undefined) {
+    return undefined;
+  }
```

Comment:
Since I want the child animation to inherit the setting from its parent I made this function return `undefined` when the setting is not defined. Then when the animation is created I can just assign `shouldReduceMotion(config.reduceMotion)` to `animation.reduceMotion` and the parent will only assign its setting to the child, when the child has it set to undefined. I will try to make this less confusing.